src/main/resources/templates/c_plus_plus/test/testUtils/junit/Junit.py

```
@@ -0,0 +1,33 @@
+from typing import Tuple, List
+from xml.etree import ElementTree as Et
+
+from testUtils.junit.TestSuite import TestSuite
+
+
+# JUnit format: https://github.com/junit-team/junit5/blob/master/platform-tests/src/test/resources/jenkins-junit.xsd
+class Junit:
+    suite: TestSuite
+    additionalSuites: List[Et.Element]
+
+    def __init__(self, suite: TestSuite, additionalSuites: List[Et.Element]) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)